ParisBNF/abb/BNFabb57.xml

```diff
@@ -189,124 +190,136 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <additions>
               <list>
                 <item xml:id="a1">
-                  <locus target="#1r"/>
-                  <desc type="OwnershipNote" xml:lang="gez"/>
+                  <locus target="#31rb"/>
+                  <desc type="OwnershipNote" xml:lang="gez">An extended ownership note, mentioning the
+                    monastery Dāgā ʾƎsṭifānos</desc>
```

Comment:
INS ID?

Reply:
I inserted ID in a quotation below, the policy is possibly to avoid tagging  inside tagging in the notes and titles, isn't it?